app/persistence/spread_repository.py

```diff
@@ -0,0 +1,165 @@
+import psycopg2
+import os
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
```

Comment:
find_dotenv() not usefull empty
remove it

Reply:
Done